SettingsView/Platforms/Android/SVItemdecoration.cs

```diff
@@ -20,6 +22,12 @@ public SVItemdecoration(Drawable drawable,SettingsView settingsView)
 
         public override void GetItemOffsets(Rect outRect, View view, RecyclerView parent, RecyclerView.State state)
         {
+            var holder = parent.GetChildViewHolder(view) as ViewHolder;
+            if (!CellIsVisible(holder?.RowInfo.Cell))
```

Comment:
@muak Maybe that is wrong for invisible sections/headers/footers also (count with offset = 1 even if invisible).

Reply:
I see. Thank you for pointing this out.